NARFO_BE/ControllerUnitTests/UnitTest1.cs

```diff
@@ -45,7 +45,7 @@ public Tests()
         public async Task GetAllMembers_ShouldReturnAllMembers()
         {
             // Create Member object
-            var newMember = new Members { Id = 1, FirstName = "Mike", LastName = " Jack", Username = "MikeJack", Password = "Password1" };
+            var newMember = new _Member { Id = 1, Firstname = "Mike", SURNAME = " Jack", Username = "MikeJack", Password = "Password1" };
```

Comment:
This line seems a bit long. Can you break it down over multiple lines? i.e.
```
var newMember = {
  Id = 1, 
  Firstname = "Mike", 
  SURNAME = " Jack",
};
```

Reply:
that is our coding style .. **thank you for your views (👍 **

Reply:
The reason I suggesting you break down over multiple lines is that it'll be easier to read and hence debug. No one likes having to scroll over long lines of code. If this truly is the coding standard you and your team agreed on, then I advise you change that "coding style". Unless you have a really good reason as to why you'd like to keep it like that, in which case I'd very much like to hear that reason.